src/services/user/hooks/userService.js

```diff
@@ -1,20 +1,11 @@
 const { authenticate } = require('@feathersjs/authentication');
 const { keep } = require('feathers-hooks-common');
-const {
-	BadRequest, Forbidden, GeneralError, NotFound,
-} = require('@feathersjs/errors');
+const { BadRequest, Forbidden, GeneralError, NotFound } = require('@feathersjs/errors');
 const logger = require('../../../logger');
 const { ObjectId } = require('../../../helper/compare');
-const {
-	hasRole,
-	hasRoleNoHook,
-	hasPermissionNoHook,
-	hasPermission,
-} = require('../../../hooks');
+const { hasRole, hasRoleNoHook, hasPermissionNoHook, hasPermission } = require('../../../hooks');
```

Comment:
Codacy found an issue: ['hasRole' is assigned a value but never used.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)